packages/core/src/routes/admin-user.ts

```diff
@@ -42,7 +43,10 @@ export default function adminUserRoutes<T extends AuthedRouter>(
 
       const user = await findUserById(userId);
 
-      ctx.body = pick(user, ...userInfoSelectFields);
+      ctx.body = {
+        ...pick(user, ...userInfoSelectFields),
+        verifications: transpileUserProfileVerifications(user),
```

Comment:
Maybe `mfaVarifications`?  Would verifications be confused with email/SMS passcode verifications?

Reply:
Should we create a `getUserProfile` library method or a util method for this? 

We have multiple endpoints returning `userProfileResponse`,  not just get user by id.

Reply:
Updated to `mfaVerifications`.

Currently, only this endpoint need this field.